operators/admin/olm-managing-custom-catalogs.adoc

```diff
@@ -7,6 +7,13 @@ toc::[]
 
 This guide describes how to work with custom catalogs for Operators packaged using either the xref:../../operators/understanding/olm-packaging-format.adoc#olm-bundle-format_olm-packaging-format[bundle format] or the legacy xref:../../operators/understanding/olm-packaging-format.adoc#olm-package-manifest-format_olm-packaging-format[package manifest format] on Operator Lifecycle Manager (OLM) in {product-title}.
 
+[IMPORTANT]
+====
```

Comment:
Friendly reminder to reproduce any changes here if you made them in the similarly reading admonition.